views/GeneralSettingsView.qml

```diff
@@ -0,0 +1,30 @@
+import QtQuick 2.0
+import QtQuick.Layouts 1.0
+import QtQuick.Controls 2.4
+
+GridLayout {
+    columns: 2
+    columnSpacing: 10
+
+    Label { text: "Number of trajectories:" }
+    IntNumField {
+        targetValue: generalSettings.numTrajectories
+        onTargetValueChanged: generalSettings.numTrajectories = targetValue
+    }
+    Label { text: "Population path:" }
+    TextField {
+        text: generalSettings.populationPath
+        onEditingFinished: generalSettings.populationPath = text
+    }
```

Comment:
it would be great if there will be an icon to choose the population path from disk (but entering free text should be possible as well)